src/main/java/it/gov/pagopa/payhub/auth/service/a2a/retreive/ClientMapper.java

```diff
@@ -0,0 +1,43 @@
+package it.gov.pagopa.payhub.auth.service.a2a.retreive;
```

Comment:
There is a typo here...
But the really problem is the package itself...
We have a packege dedicated to Mappers... move it there
```suggestion
package it.gov.pagopa.payhub.auth.service.a2a.retrieve;
```